apps/api/src/resources/swagger.yaml

```diff
@@ -980,7 +980,7 @@ components:
         update:
           description: Changes to make to the application
           type: object
-          $ref: '#/components/schemas/ApplicationContents'
+          $ref: '#/components/responses/ApplicationContentsResponse'
```

Comment:
This `ApplicationUpdates` is a request schema but the original `ApplicationContentsResponse` was in responses

Both incoming/outgoing data is using camelCase